modules/aws/test/src/smithy4s/aws/query/AwsQueryCodecSpec.scala

```diff
@@ -0,0 +1,354 @@
+/*
+ *  Copyright 2021-2022 Disney Streaming
+ *
+ *  Licensed under the Tomorrow Open Source Technology License, Version 1.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *     https://disneystreaming.github.io/TOST-1.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package smithy4s.aws.query
+
+import cats.effect.IO
+import cats.syntax.all._
+import smithy4s.ByteArray
+import smithy4s.Hints
+import smithy4s.ShapeId
+import smithy4s.schema.Schema
+import smithy4s.schema.CompilationCache
+import smithy4s.schema.Schema._
+import weaver._
+import smithy.api.XmlFlattened
+import smithy.api.XmlName
+
+object AwsQueryCodecSpec extends SimpleIOSuite {
+
+  implicit class SchemaOps[A](schema: Schema[A]) {
+    def named(name: String) = schema.withId(ShapeId("default", name))
```

Comment:
you should be able to remove this in these tests:  the name of the shape is never to be taken into consideration in this encoding.

Reply:
Done